server/events/commit_status_updater.go

```diff
@@ -51,14 +51,13 @@ func (d *DefaultCommitStatusUpdater) UpdateCombined(repo models.Repo, pull model
 	var descripWords string
 	switch status {
 	case models.PendingCommitStatus:
-		descripWords = "in progress..."
+		descripWords = genProjectStatusDescription(cmdName.String(), "in progress...")
```

Comment:
Nice work @albertorm95 !

Could you also add test(s) for this to verify the `descripWords` ?

Reply:
Sure, I can create one, just that I don't understand my current test fail :(

`--- FAIL: TestProjectOutputWrapper`